configs/bokeh.json

```diff
@@ -1,35 +1,88 @@
 {
   "index_name": "bokeh",
   "start_urls": [
-    "https://docs.bokeh.org"
+    {
+      "url": "https://docs.bokeh.org/(?P<lang>.*?)/(?P<version>.*?)/user_guide/",
+      "selectors_key": "user_guide",
+      "page_range": 3,
+      "variables": {
+        "lang": ["en"],
+        "version": ["dev", "latest", "1.3.4", "1.2.0", "1.1.0", "1.0.4", "0.12.16", "0.12.15", "0.12.14", "0.11.0"]
+      }
```

Comment:
Why not relying on the selectors as previously done?

Reply:
Because of this, you will need to update your configuration every time you release a new version

Reply:
If the issue is that the `version` selector does not extract a "latest" one, please use the DocSearch meta tags => https://community.algolia.com/docsearch/required-configuration.html#introduces-global-information-as-meta-tags

Reply:
@s-pace The initial email I got mentioned only that there was 1.3.4 indexed now, and at least the experiments I tried seemed to indicate there were no other facets other then 1.3.4 currently. I don't seem to have permission to access the actual index on the algolia account. Can you confirm if other versions have been picked up? And if not, what changes are necessary to have them be picked up? Thanks!

Reply:
Also apologies but I don't see how to apply the meta-tag to get a "latest" version. In our current scheme the files (all on S3) under "latest" are just a copy of whatever files are under the actual latest version, e.g. "1.3.4" They won't differ (by a meta tag or anything else)

Reply:
@bryevdv Could [you provide a compliant sitemap](https://community.algolia.com/docsearch/tips.html#use-a-sitemapxml)? It will help to make sure we do crawl every version

Reply:
If you are fine with filtering with the numerical version e.g. `1.3.4`, that's fine then. We do extract this value from your DOM according to the selector `.version-link:first-child a`. Make sure the facetFilter is related to this value and not `latest` since it is not present within the DOM.

Side note, we will not redirect to the `1.3.4` URLs but `latest` instead due to the meta tags `rel=canonical`

Reply:
@s-pace we currently generate `sitemap.txt` for each release, e.g 

http://docs.bokeh.org/en/1.3.4/sitemap.txt

I'm not familiar with an XML sitemap and also I am not sure how we could best keep a single "running" sitemap that has everything for all the releases offhand. We can certainly look into it but it's not something I can provide in the immediate future.

Reply:
Alright so please add these sitemaps and you will need to update your configuration at every release.

You can also define a sitemap index. [See here for reference](https://www.sitemaps.org/protocol.html#index).